content/docs/projects/README.md

```diff
@@ -9,6 +9,11 @@ functionality, and complement the core cert-manager feature-set.
 
 These tools help with security, compliance and control.
 
+- [Apply Policy to cert-manager Certificates](./approver-policy.md):
+  [approver-policy](https://github.com/cert-manager/policy-approver) is a
+  cert-manager
+  [approver](https://cert-manager.io/docs/concepts/certificaterequest/#approval)
+  that will approve or deny CertificateRequests based on CRD defined policies.
```

Comment:
I find it confusing to have the link back to https://github.com/cert-manager/policy-approver here. 
```suggestion
- [approver-policy: ](./approver-policy.md):
  a cert-manager **approver** that will approve or deny CertificateRequests based on CRD defined policies.
```

I suggest having a single link here to the new page on cert-manager.io. And in there we can mention the link to the source code and link to the approver concepts.